libs/sdk-ui-ext/src/internal/dashboardEmbedding/DashboardLayout/utils/sizing.ts

```diff
@@ -467,3 +466,47 @@ export function getDashboardLayoutWidgetDefaultHeight(widgetType: WidgetType, vi
 
     return dimension.defHeightPx;
 }
+
+export function getLayoutWithoutGridHeights<TWidget>(
+    layout: IDashboardLayout<TWidget> | IDashboardLayoutItem<TWidget>[],
+): IDashboardLayout<TWidget> {
+    if (isDashboardLayout<TWidget>(layout)) {
+        const updatedLayout: any = {
+            ...layout,
+            sections: DashboardLayoutFacade.for(layout)
+                .sections()
+                .reduce((acc: IDashboardLayoutSection<TWidget>[], section) => {
+                    if (isDashboardLayout(section.items().raw())) {
+                        return getLayoutWithoutGridHeights(section.items().raw());
```

Comment:
This looks wrong -> layout can be only in particular item widget, array of items cannot be the layout

Reply:
I changed it to pass the `widget` of the item, whenever is of layout type.